src/manifolds/SymmetricPositiveDefiniteLinearAffine.jl

```diff
@@ -369,25 +383,18 @@ and `log` the logarithmic map on [`SymmetricPositiveDefinite`](@ref)
 parallel_transport_to(::SymmetricPositiveDefinite, ::Any, ::Any, ::Any)
 
 function parallel_transport_to!(M::SymmetricPositiveDefinite{N}, Y, p, X, q) where {N}
-    distance(M, p, q) < 2 * eps(eltype(p)) && copyto!(Y, X)
-    e = eigen(Symmetric(p))
-    U = e.vectors
-    S = max.(e.values, floatmin(eltype(e.values)))
-    Ssqrt = sqrt.(S)
-    SsqrtInv = Diagonal(1 ./ Ssqrt)
-    Ssqrt = Diagonal(Ssqrt)
-    pSqrt = Symmetric(U * Ssqrt * transpose(U)) # p^1/2
-    pSqrtInv = Symmetric(U * SsqrtInv * transpose(U)) # p^(-1/2)
-    tv = Symmetric(pSqrtInv * X * pSqrtInv) # p^(-1/2)Xp^{-1/2}
-    ty = Symmetric(pSqrtInv * q * pSqrtInv) # p^(-1/2)qp^(-1/2)
+    distance(M, p, q) < 2 * eps(eltype(convert(AbstractMatrix, p))) && copyto!(Y, X)
```

Comment:
```suggestion
    distance(M, p, q) < 2 * eps(eltype(p)) && copyto!(Y, X)
```
`SPDPoint` supports `eltype`.